Source/towners.cpp

```diff
@@ -824,6 +842,7 @@ void InitTowners()
 	assert(CowCels == nullptr);
 
 	CowCels = LoadFileInMem("Towners\\Animals\\Cow.CEL");
+	BirdCels = LoadFileInMem(R"(data\bird.cl2)");

```

Comment:
what's this R lol

Reply:
A Raw string where no escape sequences are active, because windows uses backslashes for the folder devider this avoid the constant need for escaping it.
`"windows\\file\\systems\\are\\silly.txt"`
vs
`R"windows\file\systems\are\silly.txt"`

Reply:
Forward slashes are valid in the windows filesystem but MPQs will use backslashes in the name which directly get translated to a hash.